src/main.h

```diff
@@ -48,10 +48,10 @@ class CValidationState;
 struct CNodeStateStats;
 
 /** Default for -blockmaxsize and -blockminsize, which control the range of sizes the mining code will create **/
-static const unsigned int DEFAULT_BLOCK_MAX_SIZE = 750000;
+static const unsigned int DEFAULT_BLOCK_MAX_SIZE = MAX_BLOCK_SIZE;
 static const unsigned int DEFAULT_BLOCK_MIN_SIZE = 0;
 /** Default for -blockprioritysize, maximum space for zero/low-fee transactions **/
-static const unsigned int DEFAULT_BLOCK_PRIORITY_SIZE = 50000;
+static const unsigned int DEFAULT_BLOCK_PRIORITY_SIZE = DEFAULT_BLOCK_MAX_SIZE / 2;
```

Comment:
This doesn't quite match what we discussed in the meeting, which was to have a max of 1 MB of transparent-only transactions (so would be `DEFAULT_BLOCK_MAX_SIZE - 1000000` here). But given we will be changing this system again later, perhaps this is fine for now? @zookozcash @nathan-at-least?